subprojects/plugins/src/test/groovy/org/gradle/api/tasks/testing/TestTaskSpec.groovy

```diff
@@ -117,6 +117,61 @@ class TestTaskSpec extends AbstractProjectBuilderSpec {
         }
     }
 
+    def expectFirstTestSuiteFailsButSecondPasses() {
```

Comment:
Thinking some more about it, this method and the next one got me confused initially.
Why not move them below the tests?

Reply:
I tried to follow the existing structure in this test class. Above this method there are other similar methods.